logprep/input/jsonl_input.py

```diff
@@ -1,9 +1,8 @@
 """This module contains a dummy input that can be used for testing purposes."""
```

Comment:
if we compare this file with json_input.py we will see, that they only differ in one letter in line 23.
how about to merge the two input connectors into one connector? or alternativly use the magic of inheritance for this?

Reply:
That is indeed a reasonable change that should be done. I don't see it as a part of this pull-request though. This pull-request started as a bug-fix and the resulting improvements here far exceed that initial goal already. Merging these two json input types would end in a refactoring inside the logprep connectors and as we are planning to do this anyways I suggest we do it when we touch the connectors itself.